crates/typed-store/src/rocks/mod.rs

```diff
@@ -1794,6 +1794,23 @@ pub struct DBOptions {
 /// Creates a default RocksDB option, to be used when RocksDB option is not specified..
 pub fn default_db_options() -> DBOptions {
     let mut opt = rocksdb::Options::default();
+
+    // One common issue when running tests on Mac is that the default ulimit is too low,
+    // leading to I/O errors such as "Too many open files". Raising fdlimit to bypass it.
+    if let Some(limit) = fdlimit::raise_fd_limit() {
+        // on windows raise_fd_limit return None
+        opt.set_max_open_files((limit / 8) as i32);
+    }
+
+    let row_cache = rocksdb::Cache::new_lru_cache(300_000).expect("Cache is ok");
+    opt.set_row_cache(&row_cache);
```

Comment:
Even though this is how we've always been using this, I think this should move to `point_lookup_..()` because row cache is only useful for point workloads (iterators bypass this cache completely)

Reply:
TBH I think most Sui and Narwhal tables I have seen have some kind of point look up workload. I can't think of a table right now that is primarily scanned. So I'm a bit hesitant to remove a cache for lookups. The additional resource usage does not seem bad. Maybe we can change this separately?

An future alternative is to use point lookup options as the default, unless it significantly hurts scanning perf.